examples/nffPktgen/testing/minPktLoss.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018-2019 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+        "flag"
+        "fmt"
+        "os"
+        "os/signal"
+        "sync/atomic"
+        "time"
+
+        "github.com/intel-go/nff-go/examples/nffPktgen/generator"
+        "github.com/intel-go/nff-go/flow"
+        "github.com/intel-go/nff-go/packet"
+)
+
+// TEST PARAMETERS:
+const TOP_SPEED uint64 = 120000000 /*(pkt/s)*/
+const TGT_LOSS float64 = 0.005 /*(0.5% target packet loss)*/
+const TRAFFIC_DELAY uint = 3 /*(sec)*/
+
```

Comment:
Please change these constants to test command line parameters. TOP_SPEED is already present among parameters as -speed switch.